closure/testing/webdriver_test.bzl

```diff
@@ -0,0 +1,88 @@
+# Copyright 2022 The Closure Rules Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Build rule for running a webtest."""
+
+load("//closure:webfiles/web_library.bzl", "web_library")
+load("@io_bazel_rules_webtesting//web:web.bzl", "web_test_suite")
+
+def webdriver_test(
```

Comment:
a little bit od documention could be nice here. At least explaining the two main targets created that are interesting for the users: the test and the debug target (how to use invoke it with bazel)